instruqt/delivering-as-an-appliance/01-specifying-the-cluster/assignment.md

```diff
@@ -1,26 +1,224 @@
+
+```
+apiVersion: embeddedcluster.replicated.com/v1beta1
+kind: Config
+spec:
+  version: [[ Instruqt-Var key="EMBEDDED_CLUSTER_VERSION" hostname="shell" ]]
+```
+
+That specification lets the Replicated Vendor Portal know which version of
+Kubernetes to embed. There are [more options for the
+configuration](https://docs.replicated.com/reference/embedded-config), but
+that's all you need to get started.
+
+The Replicated Release
+======================
+
+The Replicated Platform distributes your software to your customers. To do
+this, it needs to know about your application, its customers, and the files
+you're shipping to them. We talk about those files as a release.
+
+Every release is built around a Helm chart, and that's all it needs. If you've
+completed the [Distributing Your Application with
+Replicated](https://play.instruqt.com/replicated/tracks/distributing-with-replicated)
+lab, you built a release around a Helm chart and installed it using Helm
+tools. To use other installation methods supported by the Replicated Platform
+you supplement the Helm chart with additional configuration files. We need a
+minimum of two configuration files: the Embedded Cluster configuration and a
+file that describes which Helm chart to install.
+
+Our Initial Appliance
+=====================
+
+To create our initial appliance, we're going to add our Helm chart and two
+configuration files to the `release` directory. These are the bare minimum set
+of files we need to create the appliance.
+
+### Bumping the Chart Version
+
+Strictly speaking, we don't have to bump the version of our Helm chart since
+we haven't changed it. It's a good practice to increment it, though, since we
+want to have consistent versions between the Helm release and the Replicated
+release. That way your customer will see a consistent version regardless of
+the install mechanism they choose.
+
+This lab uses the application [SlackerNews](https://slackernews.io) and the
+Helm sources for it are in the `slackernews` directory. Open
+`slackernews/Chart.yaml` in the "Release Editor" tab and update the `version`
+to `0.6.0`.
+
+![Updating the version of the Helm chart to 0.6.0](../assets/updating-the-chart-version.png)
+
+### Adding a Helm Chart to a Release
+
+To prepare the release, we first need to make sure our Helm chart is configured
+as part of it. From the Shell tab, package the chart and include it in the
+`release` directory, where we'll also add the additional files we need.
+
+```
+helm package -u slackernews -d release
+```
+
+We're then going to add a file that lets the Admin Console know about the Helm
+chart. It uses this file to identify which chart to install and pass the
+appropriate values. We'll look at the passing values in a later section of the
+lab when we set up the configuration screen for our application.
+
+Go to the "Release Editor" tab to add a file named `slackernews-chart.yaml` to
+your release.
```

Comment:
```suggestion
the `release` directory.
```
You've definitely implied that this is going into the `release` directory multiple times already, but figured it couldn't hurt to spell it out since this is the first time we're adding a file to the `release` dir from the Release Editor tab